tests/pytests/test_json.py

```diff
@@ -803,4 +792,28 @@ def testTagArrayLowerCase(env):
     waitForIndex(env, 'idx4')
     env.expect('FT.SEARCH', 'idx4', '@attrs:{Vivo}', 'NOCONTENT').equal([1L, 'json1'])
     env.expect('FT.SEARCH', 'idx4', '@attrs:{vivo}', 'NOCONTENT').equal([1L, 'json2'])
+
+@no_msan
+def testNullValue(env):
+    # check JSONType_Null is ignored, not failing
+    conn = getConnectionByEnv(env)
+    conn.execute_command('FT.CREATE', 'idx', 'ON', 'JSON', 'SCHEMA', '$.num', 'AS', 'num', 'NUMERIC',
```

Comment:
Suggest to test also Sortable, Separator, etc. to make sure `null` skipped values are OK